test.flux

```diff
@@ -0,0 +1,2 @@
+import "date"
```

Comment:
This is a good start but we have a method for writing this kind of test. You might want to look at `stdlib/date/minute_test.flux` as an example for how to write a unit test like this one. Any files that end in `_test.flux` and have a `test` statement inside of them will be run after doing `go generate ./stdlib; go test ./stdlib`.